src/main/webapp/instructorHelp.html

```diff
@@ -7461,7 +7568,7 @@ <h5 class="col-sm-2">
                                             Team 3
                                         </td>
                                         <td>
-                                            
+
```

Comment:
@wkurniawan07 empty lines can be indented? I think most IDEs add indentations automatically based on the previous line, so most empty lines will be created with indentations.